Tutorials/frame-and-phase-intro/reset_phase_demo.py

```diff
@@ -15,7 +14,7 @@
 from configuration import *
 from scipy.optimize import leastsq
 
-qmm = QuantumMachinesManager(port=9510)
+qmm = QuantumMachinesManager()
```

Comment:
I only wonder about this...
WDYT?

Reply:
Not about removing the port (this is needed!), but about not having IP...

Reply:
What do you want to write there? `qop_ip`

Reply:
I removed it because it was a local thing on my end and I wanted it to be general..

Reply:
I am a bit confused, so now we don't need to specify the IP at all ?

Reply:
We need to specify an IP. I'm just saying that we should have a default value inside the `QuantumMachinesManager()` such that the syntax will be correct